docs/language/src/builtins/widgets.md

```diff
@@ -462,6 +462,36 @@ export component Example inherits Window {
 }
 ```
 
+## `Switch`
+
+A `Switch` is used to toggle between `checked` and `unchecked` state represented by a boolean.
+
+### Properties
+
+-   **`checked`**: (_inout_ _bool_): Whether the switch is checked or not.
+-   **`enabled`**: (_in_ _bool_): Defaults to true. When false, the switch can't be pressed
```

Comment:
That reminds me: Shouldn't the `ProgressIndicator` also have an `enabled` property?

Style: For consistency, please specify the default at the end, like with all the other properties.

Reply:
> That reminds me: Shouldn't the ProgressIndicator also have an enabled property?

There is no `disabled` definition for `ProgressBar` in fluent and material design. I compared it also with other ui toolkits and it looks common that `ProgressBar` has no  `disabled` state

Reply:
Okay. That kind of makes sense. I guess it just shouldn't be visible as an overall element if this "disabled" state is entered in the entire UI.

One thing that's a little awkward in our gallery is that when you check the "disable widgets" checkbox (or maybe switch in the future :-), then widgets may be left "behind" and drawn still enabled because they don't support that state. That looks bad IMO.

So in this very case, I would suggest to make the visibility of the `ProgressIndicator` conditional to the enabling of all other widgets.

(I know, this is OT to this PR :-)